contrib/snapcraft/snap/snapcraft.yaml

```diff
@@ -0,0 +1,37 @@
+name: podman # you probably want to 'snapcraft register <name>'
+version: '0.11.1.1' # just for humans, typically '1.2+git' or '1.3.2'
+summary: Manage pods, containers and container images
+description: |
+  `podman` is a tool for managing Pods, Containers, and Container Images
+  in multiple formats including Docker/OSI images. It exposes the same
+  command line interface as Docker, but runs containers unprivileged by
+  default.
+
+confinement: devmode # use 'strict' once you have the right plugs and slots
+
+base: core18
+
+parts:
+  podman:
+    plugin: go
+    source: https://github.com/containers/libpod/archive/v0.11.1.1.tar.gz
+    go-importpath: github.com/containers/libpod
+    build-packages:
+     # https://github.com/containers/libpod/blob/master/install.md#build-and-run-dependencies
```

Comment:
Used this snapcraft.yaml to build it:
```
# snapcraft --debug
...

The 'podman' part is missing libraries that are not included in the snap or base. They can be satisfied by adding the following entry for this part
stage-packages:
- libarchive13
- libassuan0
- libgpgme11
- libicu60
- libostree-1-1
- libsoup2.4-1
- libxml2
'grade' property not specified: defaulting to 'stable'
Snapping 'podman' /                                                                                                                                           
Snapped podman_0.11.1.1_amd64.snap
# ls -l
total 28640
-rw-r--r--. 1 root root 29323264 Aug 12 15:28 podman_0.11.1.1_amd64.snap
-rw-r--r--. 1 root root     1134 Aug 12 15:24 snapcraft.yaml
```

Should we add the stage-packages entry there as well?

Reply:
@jnovy most likely `podman` snap won't work without them if you install and try the snap.

Reply:
Hi @jnovy, 

The 'stage-packages:' should be placed right below the 'build-packages:'

Reply:
Added the missing stage-packages to snapcraft.yaml.

Reply:
thanks for adding that @jnovy , I squashed the commits to fix the validate problem :)

Reply:
lgtm now - @mheon - Could you merge this please?